src/onnx/parse_squeeze.cpp

```diff
@@ -0,0 +1,57 @@
+#include <migraphx/onnx/op_parser.hpp>
+#include <migraphx/ranges.hpp>
+#include <migraphx/make_op.hpp>
+#include <migraphx/onnx/checks.hpp>
+#include <migraphx/instruction.hpp>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+namespace onnx {
+
+struct parse_squeeze : op_parser<parse_squeeze>
+{
+    std::vector<op_desc> operators() const
+    {
+        return {{"Squeeze", "squeeze"}, {"Unsqueeze", "unsqueeze"}};
+    }
+
+    operation assign_axes(operation& op, const std::vector<int64_t>& axes) const
+    {
+        auto v = op.to_value();
+        for(auto&& x : v)
+        {
+            if(x.get_key() == "axes")
+            {
+                x = axes;
+                break;
+            }
+        }
```

Comment:
Why not write this as `v["axes"] = axes`?

Reply:
changed